pages/create_company_page.py

```diff
@@ -0,0 +1,29 @@
+from locators import LocatorsCreateCompanyPage, LocatorsMyCompaniesPage, LocatorsVacancies
+from selenium.webdriver.common.by import By
+
+
+class CreateCompanyPage():
+    """On this page company owner can create his/her own company by filling input fields with needed
+    data snd click create"""
+
+    def __init__(self, browser):
+        self.locators = LocatorsCreateCompanyPage
+        self.locators_my_companies = LocatorsMyCompaniesPage
+        self.locators_vacancies = LocatorsVacancies
+        self.browser = browser
+
+    def enter_data(self, co_data):
+        """Enters into the specific field data"""
+        for el in range(len(self.locators.COMPANY_FIELDS)):
```

Comment:
for by, selector in self.locators.COMPANY_FIELDS:

Reply:
What do you mean?? I didn't get you

Reply:
@rvynnychuk-amplify there variable `el` is used as numbers of elements. it is used for correlation company_fields and data that will be sent to this field